tests/last_argument_expansion/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -20,17 +20,16 @@ export default function searchUsers(action$) {
     .ofType(ActionTypes.SEARCHED_USERS)
     .map(action => action.payload.query)
     .filter(q => !!q)
-    .switchMap(q =>
```

Comment:
I don't know what it should look like. Right now it alternates depending on the context. See https://github.com/jlongster/prettier/issues/625 for another example.